docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -151,8 +151,10 @@ Instead of creating new tests, leverage these two attributes to create a single
 
 Run `dotnet test`, and two of these tests fail. To make all of the tests pass, change the `if` clause at the beginning of the method:
 
+
 ```csharp
 if(candidate < 2)
```

Comment:
for the merge conflicts, you should add the identifier csharp but keep the change that @GuardRex made to the topic: 
`if (candidate < 2)` (note he added a space after the if). and also remove the extra lines added.